symplyphysics/definitions/linear_charge_density_from_charge_and_length.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## Linear charge density is the quantity of charge per unit length, at any point on a line charge distribution.
+## Charge density can be either positive or negative, since electric charge can be either positive or negative.
+
+
+## Law: λ = q / l
+## Where:
+## λ is linear charge density
+## q is charge
+## l is the length over which charge is distributed
+
+
+linear_charge_density = Symbol("linear_charge_density", units.charge / units.length)
+charge = Symbol("charge", units.charge)
+length = Symbol("length", units.length)
+
+law = Eq(linear_charge_density, charge / length)

```

Comment:
It's `definition` for definitions.
Also we usually have SI units defined for definitions. Look for a sample definition.